build.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('follow-redirects').https;
+const archiver = require('archiver');
+
+let options = {
+    host: 'uplift.shipit.staging.mozilla-releng.net',
+    path: '/coverage/supported_extensions',
+    method: 'GET',
+    headers: {"Content-Type": "text/plain"}
+}
+
+
+let req = https.get(options, (res) => {
+
+	let data = '';
+    
+	res.on('data', (chunk) => {
```

Comment:
You can remove the parentheses around `chunk`, since it's a single parameter. Also, I think you can remove the `{`:
```
res.on('data', chunk => data += chunk);
```